raiden_contracts/deploy/contract_deployer.py

```diff
@@ -77,6 +79,20 @@ def __init__(
         else:
             LOG.info("Skipped checks against the source code because it is not available.")
 
+    def _maybe_set_arbitrum_settings(self) -> None:
```

Comment:
I'm personally not a big fan of function names with the word `maybe`. This is bad for readability and understanding of the code. How about using something more generalist, like `_adjust_gas_special_case`, or even `_adjust_chain_settings`?

This way, if we have another case that needs special settings, it can be included here as well.

Reply:
✔️